### ISSUE TITLE
Update list of Geant4 data for Geant4 10.2

### ISSUE THREAD

**Issue by civanch** [cms-sw/cmsdist#2191]
Geant4 10.2 requires a new dataset G4ENSDFSTATEDATA1.2.1 (1.6Mb), tar file is available at url: http://geant4.web.cern.ch/geant4/support/download.shtml

This file may be added both to the main and DEVEL branches.

There are updated data files for 10.2 in the same page:
  G4EMLOW6.48 (24.2Mb)
  G4PhotonEvaporation3.2 (8.9 Mb)
  G4RadioactiveDecay4.3.1 (7.6 Mb)
These files are not backward compatible, so should be used only in the DEVEL branch until Geant4 10.2 will be moved to the main branch.

**Comment by ghost** [comment]
@civanch Despite the extension of the archive being .tar.gz it is not a gunzip archive.
If you compare another GEANT4 data archive that's what you see:

```
file G4EMLOW.6.48.tar.gz 
G4EMLOW.6.48.tar.gz: gzip compressed data, from Unix, last modified: Wed Oct 28 10:47:17 2015
```

while the new one says:

```
file G4ENSDFSTATE.1.2.1.tar.gz 
G4ENSDFSTATE.1.2.1.tar.gz: POSIX tar archive (GNU)
```

and have to be opened without the "z" option.
Although this is not such big of a deal, for consistency reasons should be better to have it in the same format of all the other data packages so our unpacking script doesn't need to use an ad-hoc exception.
Can you please ask the maintainer to uniform the packaging of this archive to all the others?

**Comment by civanch** [comment]
@degano , you are right, it turns out to be a bug that this data set for the 10.2p01 was not compressed. In the next patch this should be fixed. All files are compressed in the same way in the development branch. So, the situation is exceptional.

**Comment by civanch** [comment]
@degano , a new file is created http://geant4.cern.ch/support/source/G4ENSDFSTATE.1.2.2.tar.gz which has fixed format. This claimed to be fully coherent with the patch.

**Comment by ghost** [comment]
@civanch excellent. I've created the PR in #2192, which is now testing.

**Comment by ghost** [comment]
Merged in #2192
